### ISSUE TITLE
A dip to the Moonwell (9433) (BB #879)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#879]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 17.05.2015 07:06:45 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/879

<hr>

Use the robot to collect a sample of the moonwell's water.

I can summon the robot but thers no hability to collect the water.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** bewtihozzy
**Original Date:** 28.05.2015 20:14:36 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/879/#comment-18438093

<hr>

Any update on this one? 

Thanks

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 30.11.2015 16:06:58 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/879/#comment-23688093

<hr>

Fix for it, can be pushed to the repo. Resolved. // only issue is the spell here since it completes the quest even if not in the moonwell. It will go to spell issues though.

```
#!mysql

INSERT INTO `creature_template` (`entry`, `heroic_entry`, `KillCredit1`, `KillCredit2`, `modelid_A`, `modelid_A2`, `modelid_H`, `modelid_H2`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `minhealth`, `maxhealth`, `minmana`, `maxmana`, `armor`, `faction_A`, `faction_H`, `npcflag`, `speed`, `scale`, `rank`, `mindmg`, `maxdmg`, `dmgschool`, `attackpower`, `baseattacktime`, `rangeattacktime`, `unit_flags`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `class`, `race`, `minrangedmg`, `maxrangedmg`, `rangedattackpower`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `resistance1`, `resistance2`, `resistance3`, `resistance4`, `resistance5`, `resistance6`, `spell1`, `spell2`, `spell3`, `spell4`, `PetSpellDataId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `RacialLeader`, `RegenHealth`, `equipment_id`, `mechanic_immune_mask`, `flags_extra`, `ScriptName`) VALUES (17178, 0, 0, 0, 1097, 0, 1097, 0, 'Robotron 3000', '', NULL, 0, 28, 28, 853, 853, 0, 0, 1130, 104, 104, 0, 1, 1, 0, 17, 29, 0, 161, 2000, 0, 768, 0, 0, 0, 0, 0, 0, 19.6552, 25.3552, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 29992, 0, 0, 0, 9504, 0, 0, '', 0, 1, 0, 1, 0, 16384, 2, '');

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** desteny
**Original Date:** 01.12.2015 23:55:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/879/#comment-23730196

<hr>

Fix #879 thanks @Selphius

→ <<cset bffb5453f7e8>>
